src/System Application/App/Upgrade Tags/src/UpgradeTag.Codeunit.al

```diff
@@ -43,6 +43,16 @@ codeunit 9999 "Upgrade Tag"
         exit(UpgradeTagImpl.HasUpgradeTag(Tag, TagCompanyName));
     end;
 
+    /// <summary>
+    /// Verifies if the database upgrade tag exists.
+    /// </summary>
+    /// <param name="Tag">Tag code to check</param>
+    /// <returns>True if the Tag with the given code exists.</returns>
+    procedure HasDatabaseUpgradeTag(Tag: Code[250]): Boolean
```

Comment:
So the idea was to use SetUpgradeTag and HasUpgradeTag only, the code will find out from the context if the tag is per database or per company. Approving since we already have the SetDatabaseUpgradeTag, which is wrongly named IMO. It should be PerDatabaseUpgradeTag